src/Forside/barnehage/SammenligningspanelBarnehage/SammenligningspanelBarnehage.tsx

```diff
@@ -20,6 +23,7 @@ export const SammenligningspanelBarnehage: FunctionComponent<{
             )}
             <div className="sammenligningspanel-barnehage" ref={panelRef}>
                 <ReactToPrint
+                    onBeforePrint={() => sendEvent('forside barnehage', 'printet')}
```

Comment:
hva med å bruke `onAfterPrint` i stedet? 
 `onAfterPrint={() => sendEvent('forside barnehage', 'printet')}`

Reply:
onAfterPrint gir ingen garanti for at det faktisk er printet - bare at dialogboksen er lukket. Fordi f.eks. strømmen kan gå etter bruker har klikket men før dialogboksen har lukket seg, tenkte jeg at onBeforePrint gir bedre datakvalitet

Reply:
Enig, da fanger vi i allefall når knappen er tatt i bruk 👍